tools/templates/install_templates.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+#
+# Copyright 2023 New Vector Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+echo "Zipping the contents of the 'files' directory..."
+
+rm -f ./tmp/file_templates.zip
```

Comment:
If `tmp` dir doesn't exist, the script fails. Maybe it needs an extra check.

Reply:
89c2130c1d28212d3023b8160398e5e7a832b3b0